docs/source/specification.md

```diff
@@ -72,13 +72,100 @@ Below we describe each level of the `rawdata` folder hierarchy in more detail. T
 
 ### Datatype
 
-The following datatypes are supported:
+The datatype folder, placed in the session level folder, is where
+data is stored. Two sets of datatype folder names are supported,
+either '*Broad*' or '*Narrow*'. The *Broad* datatype names are designed to
+cover most use cases:
 
 * `ephys`: electrophysiology (e.g. Neuropixel probes, tetrodes)
 * `behav`: behavioural (e.g. video and audio files, response logs)
 * `funcimg`: functional imaging (e.g. calcium and voltage imaging)
 * `anat`: anatomical (e.g. histology, using confocal or lightsheet)
 
+In some cases, the *Broad* datatype names may not be specific enough,
+for example if two different types of electrophysiological (`ephys`)
+recording were run. In this case, the *Broad* datatype
+name *must* be substituted for a *Narrow* datatype name (rather than placing two
+different datatypes in a *Broad* datatype folder). See the dropdown below for
+the full list of supported *Narrow* datatypes.
+
+:::{dropdown} Narrow datatypes
+:color: info
+:icon: info
+
+If a *Broad* datatype is replaced with a corresponding *Narrow* datatype for any
+session within a particular subject, the *Broad* datatype *must* not be used for
+that subject. It is not then necessary to use the *Narrow* datatype
```

Comment:
Hey @niksirbi you are right here, allowing this can only lead to problems and you don't gain much from it. Is the below clear?

```
If a *Narrow* datatype is used instead of a *Broad* datatype anywhere
in the project, the *Narrow* datatype *must* be used across the entire 
project. The *Broad* datatype for that category *must* no longer be used.
```

Reply:
Yes, I'm happy with that.